src/Sylius/Bundle/PricingBundle/Form/EventListener/BuildPriceableFormSubscriber.php

```diff
@@ -85,8 +86,13 @@ public function preBind(FormEvent $event)
      * @param string        $calculatorType
      * @param array         $data
      */
-    protected function addConfigurationFields(FormInterface $form, $calculatorType, array $data = array())
+    protected function addConfigurationFields(FormInterface $form, $calculatorType = null, array $data = array())
```

Comment:
Hmmm, IMO `pricingCalculator` should never be `null`, if there is any case that set it to `null` it should be replaced with that `Calculators::STANDARD` value.

@pjedrzejewski @michalmarcinkowski Agree?